docs/kits.rst

```diff
@@ -12,22 +32,58 @@ CMake Kits
 How Are Kits Found and Defined?
 ===============================
 
-Upon first startup, CMake Tools will scan the system for available toolchains.
-It looks in certain directories for the presence of compilers or Visual Studio
-installations (using ``vswhere``) to populate the initial list of Kits.
+Upon first startup, CMake Tools will :ref:`scan the system <kit-scan>` for
+available toolchains. It looks in certain directories for the presence of
+compilers or Visual Studio installations (using ``vswhere``) to populate the
+initial list of Kits.
+
+.. _user-local-kits:
+
+User-Local Kits
+***************
 
-The list of kits is stored in a user-local file, which you can edit by invoking
-*Edit CMake Kits* from the command palette. It will open the ``cmake-kits.json``
-file:
+User-local kits are kits that are available to a particular user for all
+projects open with CMake Tools.
+
+The user-local list of kits is stored in a user-owned file, which you can edit
```

Comment:
maybe add the path where it is stored, so that the user know that it is stored in the profile folder.

Reply:
Maybe? I'm wary of publicly documenting where this file is stored, as it's path is more of an implementation detail. That's why there is a dedicated "Edit Kits" command: So the user doesn't need to know where it is.

Reply:
One of the questions on issue reporting was "where is the kit file?". 
But you are right it is acceeable by "edit kit".